data/mods/more_classes_scenarios/cs_classes.json

```diff
@@ -322,43 +318,37 @@
     "ident": "bio_gangster",
     "name": "Bionic Gangster",
     "description": "You were the boss' favorite, their protege; they always counted on you to get the toughest jobs done. Seeing your potential, they invested in \"basic\" augments and the best gear on the market to better aid you in your job. After enjoying some period of freedom to do as you wanted, now you find yourself needing those skills to survive. ",
-    "points": 10,
+    "points": 8,
     "CBMs": [
       "bio_targeting",
-      "bio_sunglasses",
-      "bio_ears",
-      "bio_carbon",
-      "bio_blade",
       "bio_shock",
+      "bio_carbon",
+      "bio_adrenaline",
       "bio_night_vision",
-      "bio_torsionratchet",
+      "bio_face_mask",
+      "bio_metabolics",
       "bio_power_storage_mkII"
     ],
```

Comment:
I would say the gangster's previous set of bionics was quite flavourful. What's the rationale in replacing it with an assassin's set of bionics? This is someone geared towards intimidation and thuggery, finger blades and sunglasses make sense.

Reply:
It was always meant to be a fusion of gangster and bionic assassin but when creating it I added way more than that and justified it with a higher point cost and locking it to a specific scenario.